spec/result.json

```diff
@@ -0,0 +1,105 @@
+    "required": [
+        "head"
+    ],
+    "dependencies": {
+        "boolean": {
+            "not": {
+                "required": [
+                    "results"
+                ]
+            }
+        },
+        "results": {
+            "not": {
+                "required": [
+                    "boolean"
+                ]
+            }
+        }
+    },
+    "properties": {
+        "head": {
+            "type": "object",
+            "properties": {
+                "additionalProperties": false,
+                "link": {
+                    "type": "array",
+                    "items": {
+                        "type": "string",
+                        "format": "uri"
+                    }
+                },
+                "vars": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                }
+            }
+        },
+        "boolean": {
+            "type": "boolean"
+        },
+        "results": {
+            "type": "object",
+            "required": [
+                "bindings"
+            ],
+            "properties": {
+                "bindings": {
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "additionalProperties": {
+                            "type": "object",
+                            "dependencies": {
+                                "xml:lang": {
+                                    "not": {
+                                        "required": [
+                                            "datatype"
+                                        ]
+                                    }
+                                },
+                                "datatype": {
+                                    "not": {
+                                        "required": [
+                                            "xml:lang"
+                                        ]
+                                    }
+                                }
+                            },
+                            "properties": {
+                                "type": {
+                                    "type": "string",
+                                    "enum": [
+                                        "uri",
+                                        "literal",
+                                        "bnode"
+                                    ]
+                                },
+                                "value": {
+                                    "type": "string"
+                                },
+                                "xml:lang": {
+                                    "type": "string"
+                                },
+                                "datatype": {
+                                    "type": "string",
+                                    "format": "uri"
```

Comment:
I believe it should be `"iri"` and not `"uri"`. https://json-schema.org/understanding-json-schema/reference/string.html#uri-template